scripts/autoload/Global.gd

```diff
@@ -3,4 +3,4 @@ extends Node
 signal crew_assigned(crew: CrewMember)
 signal crew_selected(crew: CrewMember)
 
-var station: Station = preload ("res://assets/station/station_resources.tres")
+var station: Station = load("res://assets/station/station_resources.tres")
```

Comment:
I read that preload should be use when ' when you require resources to be available immediately and consistently'. In this case are you using load because the resources are 'rarely' changed?

Reply:
hard to explain but there were issues with preloading the station resources if they referenced classes that themselves had references to the station variable, i think technically we should be able to find a way around it if it becomes a problem/ is not efficient enough